pkg/controller/podautoscaler/horizontal.go

```diff
@@ -106,7 +106,7 @@ func newInformer(controller *HorizontalController, resyncPeriod time.Duration) (
 
 func NewHorizontalController(evtNamespacer unversionedcore.EventsGetter, scaleNamespacer unversionedextensions.ScalesGetter, hpaNamespacer unversionedautoscaling.HorizontalPodAutoscalersGetter, metricsClient metrics.MetricsClient, resyncPeriod time.Duration) *HorizontalController {
 	broadcaster := record.NewBroadcaster()
-	broadcaster.StartRecordingToSink(&unversionedcore.EventSinkImpl{Interface: evtNamespacer.Events("")})
+	broadcaster.StartRecordingToSink(evtNamespacer.Events(""))
```

Comment:
strange name: namespacer. Any reason why it's not called client here? (yes, not really part of the PR).